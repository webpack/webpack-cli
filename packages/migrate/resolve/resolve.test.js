"use strict";

const defineTest = require("@webpack-cli/utils//defineTest");

defineTest(__dirname, "resolve");
