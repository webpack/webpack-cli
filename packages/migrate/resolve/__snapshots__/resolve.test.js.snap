// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`resolve transforms correctly 1`] = `
"import path from 'path';

export default [{
    resolve: {
        modules: [path.resolve('/src')]
    }
}, {
    resolve: {
        modules: [path.resolve('/src')]
    }
}, {
    resolve: {
        modules: [path.resolve('/src'), 'node_modules']
    }
}, {
    resolve: {
        modules: ['node_modules', path.resolve('/src')]
    }
}];
"
`;
